dt-core/admin/admin-settings-endpoints.php

```diff
@@ -229,13 +229,23 @@ public function update_languages( WP_REST_REQUEST $request ) {
         $params = $request->get_params();
         $saved_language_options = get_option( 'dt_working_languages', [] );
 
+        function recursive_array_update( &$target, $source ) {
+            foreach ( $source as $key => $value ) {
+                if ( is_array( $value ) && isset( $target[$key] ) && is_array( $target[$key] ) ) {
+                    recursive_array_update( $target[$key], $value );
+                } elseif ( !isset( $target[$key] ) || $target[$key] !== $value ) {
+                    $target[$key] = $value;
+                }
+            }
+        }
         /**
          * @todo:
          * only save user provided customizations to the dt_working_languages options
          * this includes labels, the iso code, the enabled status and translations
          * does not include: the key, default labels etc
          */
 
+        recursive_array_update( $saved_language_options, $params );
         update_option( 'dt_working_languages', $saved_language_options, false );
```

Comment:
@CptHappyHands 
Here we only want to save the custom values the admin has entered. Not any that are default.

So if the admin sets english to enabled:
We'd do `saved_language_options[en_US]['enabled'] = true`

I expect a lot of the code from process_languages_box() would be copied here.